backend/code/archipelag/market/views.py

```diff
@@ -23,22 +24,41 @@ def get(self, request):
 @login_required
 def market_create(request):
     if request.method == 'POST':
-        body_unicode = request.body.decode('utf-8')
-        body_data = loads(body_unicode)
-
-        new_market = Market.objects.create(
-            owner=request.user.ngouser,
-            title=body_data["title"],
-            url=body_data["url"],
-            date_starting=body_data["date_starting"],
-            date_ending=body_data["date_ending"],
-            hashtag = body_data["hashtag"]
-        )
-        data = dict(url='/message/create/%s'%new_market.id)
-        return JsonResponse(data)
+        body_data = loads(request.body.decode('utf-8'))
```

Comment:
Too dangerous to put not validated data. Could you use django forms to validate input?

Reply:
added